batchflow/batch.py

```diff
@@ -175,14 +174,14 @@ def _make_batch(data):
             if batch_size is None:
                 new_comp = [b.get(component=comp) for b in batches[:break_point]]
             else:
-                b = batches[break_point]
-                last_batch_len_ = b.get_pos(None, comp, b.indices[last_batch_len - 1])
+                last_batch = batches[break_point]
+                last_batch_last_index = last_batch.get_pos(None, comp, last_batch.indices[last_batch_len - 1])
                 new_comp = [b.get(component=comp) for b in batches[:break_point]] + \
```

Comment:
these are just readability improvements to get rid of variable `b` both inside and outside of list comprehensions